suitcase/nexus.py

```diff
@@ -171,7 +170,7 @@ def export(headers, filename,
                 datetime_string = time.asctime(time.gmtime(start))
                 ds.attrs['start'] = dateutil.parser.parse(datetime_string).isoformat()
 
-                [fill_event(e) for e in events]
+                [db.fill_event(e) for e in events]
```

Comment:
I believe that `db.fill_events(events)` would work here, and `db.fill_event` is deprecated (if I recall correctly) or at least discouraged.

Reply:
I will add fill=True at db.get_events.

Reply:
yes, i think that looks good, please add `fill=True`.

Reply:
thanks. I only made change in the hdf file, forget to do it at nexus file.